index.js

```diff
@@ -94,15 +99,49 @@ const getAddresses = () => {
   return _rlAddressToRequests
 }
 
+
+const hash = (address) => {
+  return Crypto.SHA256(address).toString(Crypto.enc.Hex);
+
+}
+
 const setTotalActiveUsers = (req) => {
-  const address = req.connection.remoteAddress
-  const lastRequestAt = _rlAddressToRequests[address] ? _rlAddressToRequests[address].lastRequestAt : null
+  const address = hash(req.connection.remoteAddress)
+  incrementActiveUsers(address)
```

Comment:
Do incrementActiveUsers really need to be a separate method? It is only used once and the function itself is quite shorts.

Reply:
Well not currently no. But we figured that it looked a bit nicer.

Reply:
Removed functions